tests/test_unit.py

```diff
@@ -5148,61 +4990,33 @@ def getNameTester(s, l, t):
             self.assertEqual(len(stmt[1, ...]('tests').parseString('test test').tests), 2, "multipled(3) failure with setResultsName")
             self.assertEqual(len(stmt[2, ...]('tests').parseString('test test').tests), 2, "multipled(3) failure with setResultsName")
 
-def makeTestSuite():
-    import inspect
-    suite = TestSuite()
-    suite.addTest(PyparsingTestInit())
-
-    test_case_classes = ParseTestCase.__subclasses__()
-    # put classes in order as they are listed in the source code
-    test_case_classes.sort(key=lambda cls: inspect.getsourcelines(cls)[1])
 
-    test_case_classes.remove(PyparsingTestInit)
-    # test_case_classes.remove(ParseASMLTest)
-    if IRON_PYTHON_ENV:
-        test_case_classes.remove(OriginalTextForTest)
```

Comment:
I am not sure why this test was excluded.  If it still needs to be then we can annotate the method

        @skipIf(IRON_PYTHON_ENV, "not expected to work in iron python")

Reply:
At one time, the behavior in this test did not run correctly on IronPython. I don't know if this is still an issue or not.

Reply:
I suggest we do not skip the test.  If/when it fails in IronPython, then we have something to fix